common/src/main/java/com/wynntils/services/map/type/ServiceKind.java

```diff
@@ -32,7 +32,9 @@ public enum ServiceKind {
     TRADE_MARKET("Trade Market", Texture.TRADE_MARKET, "trade-market"),
     WEAPON_MERCHANT("Weapon Merchant", Texture.WEAPON_MERCHANT, "merchant:weapon"),
     WEAPONSMITHING_STATION("Weaponsmithing Station", Texture.WEAPONSMITHING_STATION, "profession:weaponsmithing"),
-    WOODWORKING_STATION("Woodworking Station", Texture.WOODWORKING_STATION, "profession:woodworking");
+    WOODWORKING_STATION("Woodworking Station", Texture.WOODWORKING_STATION, "profession:woodworking"),
+    MERCHANT("Merchant", null, "merchant"),
+    STATION("Station", null, "profession");
```

Comment:
What's this change?

Reply:
Basically it was for testing that you could do `/location service merchant` and get a list of all merchants. I'm not 100% convinced it is useful, so we can revert this.

Reply:
If the "kind" doesn't exist in the services file we provide, then it should not be kept (for now, at least).